src/Row.ts

```diff
@@ -0,0 +1,45 @@
+import { WidgetProperties } from '@dojo/widget-core/interfaces';
+import WidgetBase from '@dojo/widget-core/WidgetBase';
+import { RegistryMixin, RegistryMixinProperties } from '@dojo/widget-core/mixins/Registry';
+import { v, w } from '@dojo/widget-core/d';
+import { HasColumns } from './interfaces';
+import { CellProperties } from './Cell';
+import { theme, ThemeableMixin } from '@dojo/widget-core/mixins/Themeable';
+
+import * as rowClasses from './styles/row.css';
+
+export interface RowProperties extends WidgetProperties, HasColumns, RegistryMixinProperties {
+	item: any;
+}
+
+@theme(rowClasses)
+class Row  extends ThemeableMixin(RegistryMixin(WidgetBase))<RowProperties> {
```

Comment:
When using mixins we need to declare the base type as a `const` before using it to extend.

```ts
export type RowBase = ThemeableMixin(RegistryMixin(WidgetBase));

class Row extends RowBase<RowProperties> {
     // .....
}
```

Also there is an extra space.